### PR TITLE
Minor DataTexture3D Clean up

### PR DESCRIPTION
**Pull request by takahirox**
Commented inline...

### CODE REVIEW

File: src/renderers/webgl/WebGLUniforms.js [hunk 1]
```diff
@@ -476,7 +476,7 @@ function getSingularSetter( type ) {
 		case 0x8b5c: return setValue4fm; // _MAT4
 
 		case 0x8b5e: case 0x8d66: return setValueT1; // SAMPLER_2D, SAMPLER_EXTERNAL_OES
-		case 0x8B5F: return setValueT3D1; // SAMPLER_3D
+		case 0x8b5f: return setValueT3D1; // SAMPLER_3D
```
Comment (takahirox) [inline src/renderers/webgl/WebGLUniforms.js]: Consistent code style, the numbers are lowercase here.

File: src/textures/DataTexture3D.d.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export class DataTexture3D extends Texture {
   constructor(
     data: ArrayBuffer | TypedArray,
     width: number,
-    heighht: number,
```
Comment (takahirox) [inline src/textures/DataTexture3D.d.ts]: typo

File: src/textures/DataTexture3D.d.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@ export class DataTexture3D extends Texture {
...
-    depth?: number
```
Comment (takahirox) [inline src/textures/DataTexture3D.d.ts]: I don't think `depth` is optional.

File: src/renderers/WebGLRenderer.js [hunk 4]
```diff
@@ -2462,16 +2462,11 @@ function WebGLRenderer( parameters ) {
...
-		// backwards compatibility: peel texture.texture
-		return function setTexture3D( texture, slot ) {
-
-			textures.setTexture3D( texture, slot );
-
-		};
+		textures.setTexture3D( texture, slot );
 
-	}() );
+	};
```
Comment (takahirox) [inline src/renderers/WebGLRenderer.js]: I don't see any special reasons to use IIFE here.

File: src/textures/DataTexture3D.d.ts [hunk 5]
```diff
@@ -6,7 +6,7 @@ export class DataTexture3D extends Texture {
...
+    height: number,
+    depth: number
```
Comment (takahirox) [inline src/textures/DataTexture3D.d.ts]: 1. Typo
2. I don't think `depth` is optional

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**